docs/FAQ.md

```diff
@@ -55,6 +58,57 @@ After some time building, I see the VehicleAI window stuck at 95%.
 
 monoDrive's Scenario Editor has high quality graphics that will take some time to compile when opening for the first time. Verify the process is still running by opening the Task Manager and see shaders are still compiling.
 <p>&nbsp;</p>
+
+##### Generate Project Files option is missing 
+
+<div class="img_container">
+   <img class='lg_img' src="../imgs/faq_generate.png"/>
+</div>
+
+You may need to permanently associate the project with Unreal Engine by opening the .uproject first before generating the project files. Then after rebooting, the "generate project files" may show as an option then. If this is not the case, try this solution offered from Unreal Engine Support:
+
+Set the file association of the “.uproject” file to “C:\Program Files (x86)\Epic Games\Launcher\Engine\Binaries\Win64\UnrealVersionSelector.exe” by following these steps:
+1. right-click the .uproject file
+1. click “open with…”
+1. click “choose another app”
+1. click “more apps”
+1. click “look for another app”
+1. browse to “C:\Program Files (x86)\Epic Games\Launcher\Engine\Binaries\Win64”
+1. select “UnrealVersionSelector.exe”
+1. click “open”
```

Comment:
This list is not rendering correctly: 
![image](https://user-images.githubusercontent.com/6026642/173871265-ed7e1a14-6daf-4ff1-8278-819bf70bd100.png)